src/cli/format.js

```diff
@@ -252,14 +253,20 @@ async function formatStdin(context) {
   const relativeFilepath = context.argv.ignorePath
     ? path.relative(path.dirname(context.argv.ignorePath), filepath)
     : path.relative(process.cwd(), filepath);
-
   try {
-    const input = await getStdin();
-
+    let input;
+    if (context.argv.stdin) {
+      printToScreen("Start writing your code snippet below");
+      printToScreen("After finishing press ctrl+D to exit the read mode");
+      input = await getInputFromTerminal();
```

Comment:
I'm not near a laptop, will <kbd>Ctrl + C</kbd> reject the Promise? Do we need handle it?

Reply:
Do not think we need to handle that scenario. 
Pressing `Ctrl + C` exits the read mode without any error.

Reply:
https://user-images.githubusercontent.com/22116082/190865506-23ce198d-4c84-4faf-b411-d72d376f7d54.mov

Reply:
👍